### ISSUE TITLE
add docs page about contributing; include information about various repos

### ISSUE THREAD

**Issue by susanev** [pulumi/pulumi-hugo#1343]
someone searched for pulumi issues today using the docs search.
im unsure what they were looking for, but this made me realize that we don't really have a docs page about contributing beyond https://www.pulumi.com/docs/guides/pulumi-packages/contribute-to-pulumi-registry/

might be neat to have a general contributing docs page
some things to consider including..
- link to code of conduct
- information about various repos, and what things live in them
- link to public roadmap
- information about whats generated vs hand-authored and how-to tell

i know a lot of this is available in the readmes, but this would be more high-level

may possibly overlap with https://github.com/pulumi/home/issues/1895

**Comment by susanev** [comment]
@cnunciato @guineveresaenger what do yall think? does this sound like a good idea? id be happy to get something started if that would be helpful, also happy to leave it for someone else.

**Comment by cnunciato** [comment]
@susanev Yes -- this is exactly what I had in mind for the content-development guide. The doc I put together with the questions -- we should make sure these kinds of questions get into that doc, and from there, we'll produce a publicly accessible guide that'll answer all of these questions in one place..

**Comment by susanev** [comment]
okay cool, but is that work planned to happen soon? i wouldn't want this to wait until you launch what seems like a massive project because i think we will lose out on contributions if we wait too long.
